test/functional/p2p_erlay.py

```diff
@@ -0,0 +1,555 @@
+#!/usr/bin/env python3
+# Copyhigh (c) 2016-2017 The Bitcoin Core developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+"""Test reconciliation-based transaction relay protocol."""
+
+from io import BytesIO
+from test_framework.siphash import siphash256
+import random
+import struct
+import time
+
+from test_framework.key import TaggedHash
+from test_framework.messages import (
+    msg_inv, msg_getdata, msg_wtxidrelay,
+    msg_verack, msg_sendrecon, msg_reqrecon,
+    msg_sketch, msg_reqsketchext, msg_reconcildiff,
+    MSG_WTX, CTransaction, CInv,
+)
+from test_framework.p2p import P2PDataStore, MIN_P2P_VERSION_SUPPORTED
```

Comment:
linter turned up:
`test/functional/p2p_erlay.py:20:1: F401 'test_framework.p2p.MIN_P2P_VERSION_SUPPORTED' imported but unused`

Reply:
oops, sorry I keep missing these issues. Linter broke on my machine for some reason, I better fix it before pushing again.